cmd/cluster/context.go

```diff
@@ -601,28 +619,38 @@ func GetPagerdutyClient(usertoken string, oauthtoken string) (*pd.Client, error)
 	if client != nil {
 		return client, err
 	}
+
 	client, err = getPDOauthClient(oauthtoken)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create both user and oauth clients for pd")
+		return nil, fmt.Errorf("failed to create both user and oauth clients for pd, neither key pd_oauth_token or pd_user_token are set in config file")
 	}
 	return client, err
 }
 
-func GetPDSeviceID(baseDomain string, usertoken string, oauthtoken string) (string, error) {
+func GetPDSeviceID(baseDomain string, usertoken string, oauthtoken string, team_ids []string) ([]string, error) {
```

Comment:
With regards to your comment in the story:
```
After some troubleshooting, I found that the latest version of osdctl changed how cluster context is handled and now requires that  pd_oauth_token, pd_user_token, and jira_token are either passed as parameters or informed in the osdctl config file.
```

Can we add an initial check to this function with something like this (exact error message up for debate :D)? Would that let `osdctl cluster context` work without specifying either of these and just print an informational message about how the user could get PagerDuty information if they wanted?

```go
if usertoken == "" || oauthtoken == "" {
  return nil, errors.New("--usertoken and --oauthtoken must be specified to view PagerDuty information")
}
```